### DISCUSSION TITLE
Can Resolvers be chained?

### DISCUSSION THREAD

**Discussion by quinnjr** [cloudreach/sceptre#1316] [Q&A] [answered]
Hello,

I'm currently writing a stack where I'd like for two resolvers to be chained together to determine the final value sent to the template. The concept is essentially

```
S3ObjectVersion: !get_most_recent_s3_version_resolver !stack_output_external {{ stack_group_config.stack_prefix }}
```

But I can't seem to get the values I desire.

Is it not possible to chain `!stack_output_external` to return a value that then gets consumed by my custom resolver `!get_most_recent_s3_version_resolver`?

**Comment by jfalkenstein** [comment, marked as answer]
Take a look at this PR: https://github.com/Sceptre/sceptre/pull/1313. It's coming!
